### ISSUE TITLE
At certain zoom levels map can fail WCAG 2.5.8 due to pins overlapping

### ISSUE THREAD

**Issue by Llewington** [Leaflet/Leaflet#10191]
The issue relates to the [Web Content Accessibility Guidelines (WCAG) Success Criterion 2.5.8 Target Size (Minimum)](https://w3c.github.io/wcag/understanding/target-size-minimum.html) which regulates the size and spacing of input controls on webpages. Essentially it states interactive (or targetable) elements must not overlap others to be considered having sufficient spacing.

I have a test map where the pins overlap when zoomed out and fail that criterion. https://uhi.ac.uk/en/dev/leaflet-test

I don't think even Google Maps has fully tackled this issue google maps hasn’t even fully fixed the issue but they somewhat tackle it by merging pins together when zoomed out enough and presenting a list of the amalgamated pin data if clicked or by just not showing some stuff until zoomed in to a certain level.

**Comment by mourner** [comment]
This is out of scope for Leaflet core. You could take a look at https://github.com/leaflet/leaflet.markercluster
